### PR TITLE
Fix rubocop UnreachableCode violations

### PR DESCRIPTION
**Pull request by pablo-code-org**
There are 2 violations of this type produced by unused/disabled scripts.
Disabling using # rubocop:disable Lint/UnreachableCode

## Testing story

commands used:
`bundle exec rubocop `
 bundle exec rubocop --only Lint/UnreachableCode
For debugging:
 bundle exec rubocop -d    

```
pablom@pablos-air code-dot-org % bundle exec rubocop --only Lint/UnreachableCode                 
Inspecting 3124 files
....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................

3124 files inspected, no offenses detected

```

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [x] New features are translatable or updates will not break translations
- [x] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [x] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: bin/circle/merge-eyes-baselines [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 require_relative '../../lib/cdo/git_utils'
 require_relative '../../lib/cdo/eyes_utils'
-
 # Temporarily disable all eyes branch merging
-exit(0)
+enabled = false
```
Comment (cat5inthecradle) [inline bin/circle/merge-eyes-baselines]: is the original code technically 'enabled = true' in both of these examples? This PR appears to result in NOT running the original code in both cases.

Comment (pablo-code-org) [reply bin/circle/merge-eyes-baselines]: yeah, in the original code, this scripts were disabled.
The both had an "exit(0)" which made the rest of the code unreachable and therefor throwing that lint warning.
the intention is to still to NOT run this code. 

This is kind of a hacky way to disabling through code and avoid that warning.

The options were:
- Remove the code (but that would made us lose the ability to enable them easily
- Add a huge comment block. This throws a warning

Comment (cat5inthecradle) [reply bin/circle/merge-eyes-baselines]: ah, got it!

Comment (Hamms) [reply bin/circle/merge-eyes-baselines]: We do want to preserve existing behavior here. I agree that we don't want to either remove the code or comment it out, but I don't think we should change any functionality in pursuit of these fixes.

We've already got a comment and a puts statement explaining to developers why we are intentionally leaving this code inaccessible, I think we should just make sure we also give that explanation in a way rubocop can also understand:

```ruby
# rubocop:disable Lint/UnreachableCode
```

Comment (Hamms) [reply bin/circle/merge-eyes-baselines]: See https://github.com/code-dot-org/code-dot-org/pull/47117 for an example of what _removing_ a `rubocop:disable` looks like (also, I wouldn't mind a review on that if you have a chance! :smile: )

**Comment by pablo-code-org** [comment]
I need to fix the merge conflicts :(

**Comment by Hamms** [comment]
Easiest way to fix the merge conflicts might be to simply rerun `rubocop --auto-gen-config --exclude-limit 0`

**Comment by pablo-code-org** [comment]
I am so confused if I still have merge conflicts or not. 

On the commit tabs it says it is good, but here it says some checks were not successful and cannot access the details (unauthorized error). 

Not sure if what I am seeing is an old message

**Comment by Hamms** [comment]
The merge conflict that you're seeing here on the pull request is valid. What you'll want to do do resolve is to make sure you've locally pulled from staging latest, then merge staging into this branch. Git should prompt you to fix conflicts in `.rubocop_todo.yml`, at which point you should either go in and do so manually or run `rubocop --auto-gen-config --exclude-limit 0` to do so automatically:

```
[~/code-dot-org (fix_rubocop_Unreachable_code)]$ git merge staging
Auto-merging .rubocop_todo.yml
CONFLICT (content): Merge conflict in .rubocop_todo.yml
Automatic merge failed; fix conflicts and then commit the result.
[~/code-dot-org (fix_rubocop_Unreachable_code|MERGING)]$ 
```

 You can then `git add` the fixed file and run `git merge --continue` to finish the merge locally. You should then be able to `git push` to resolve the merge here on github.

**Pull request closed without merging**